stanford/stage-ssh-config

```diff
@@ -1,20 +1,8 @@
-#### edx-stage VPC 
-
-Host 54.241.183.3
-#Host vpc-jumpbox
-    HostName 54.241.183.3
-    User ubuntu
-    ForwardAgent yes
-
-Host *.us-west-1.compute.internal
-    User ubuntu
-    ForwardAgent yes
-    ProxyCommand ssh -W %h:%p ubuntu@54.241.183.3
-
 Host *
     ForwardAgent yes
     SendEnv LANG LC_*
     HashKnownHosts yes
     GSSAPIAuthentication yes
     GSSAPIDelegateCredentials no
-
+    User ubuntu
```

Comment:
I think we can drop this line, in favor of the definition in `playbooks/ansible.cfg`.